src/components/Span/Span.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Span = ({ content, ...otherProps }) => (
```

Comment:
I think we might need to create in components another folder called UI, and put all these small items, such as Button, Error, Span, FormInput inside there, for better clarity. What do you think?

Reply:
Good idea!

UI is a big name, no? all components are UI...

maybe "units" in lower case for distinguish it's not a component folder?

Reply:
Yes, that works.